setup.py

```diff
@@ -1,23 +1,20 @@
 from setuptools import setup, find_packages
-from io import open
 
 version = "2.6.2.dev0"
 
-with open("README.rst", "rt", encoding="UTF-8") as file:
+with open("README.rst", encoding="UTF-8") as file:
     long_description = file.read() + "\n"
 
-with open("HISTORY.txt", "rt", encoding="UTF-8") as file:
+with open("HISTORY.txt", encoding="UTF-8") as file:
     long_description += file.read()
 
 setup(
     name="pyroma",
     version=version,
     description="Test your project's packaging friendliness",
     long_description=long_description,
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
+    python_requires=">=3.7",
```

Comment:
Not sure about updating this one - this would cause pip to skip this version with e.g. Python 3.6. It was untouched in e517709c4fb63409d0ee69702095233bfb15474d, but I'm not sure if this was deliberate.

Also it might be a bit early to really drop 3.6? It's still [supported](https://devguide.python.org/#status-of-python-branches) until the end of the year, and probably still somewhat widely used because of [Ubuntu 18.04 LTS](https://packages.ubuntu.com/bionic/python3) (superseded by 20.04, but supported by Canonical until 2023, so I suppose many people didn't upgrade yet).